os/board/rtl8730e/src/component/bluetooth/example/ble_scatternet/scatternet.c

```diff
@@ -287,6 +293,21 @@ static rtk_bt_evt_cb_ret_t ble_tizenrt_scatternet_gap_app_callback(uint8_t evt_c
 		memcpy(ble_tizenrt_scatternet_conn_ind, conn_ind, sizeof(rtk_bt_le_conn_ind_t));
         rtk_bt_le_addr_to_str(&(conn_ind->peer_addr), le_addr, sizeof(le_addr));
         if (!conn_ind->err) {
+#ifdef CONFIG_PM
+            int ret;
+            int fd = open(PM_LOCK_PATH, O_WRONLY);
+            if (fd < 0) {
+                    pmdbg("Failed to lock PM state transition, fd: %d\n", fd);
+            } else {
+		    /* We lock PM in PM_NORMAL state for 10 minutes to allow comfortable onboarding */
+                    ret = write(fd, NULL, 600000000);
+                    if (ret < 0) {
+                            pmdbg("Failed to write via procfs to lock PM, ret:  %d\n", ret);
+                    }
+                    pmvdbg("PM state transition is locked\n");
+            }
+            close(fd);
+#endif
```

Comment:
Looks good, but one thing to confirm. In os_start, PM_NORMAL will be locked once, did we unlock once first before starting wifi and bt? If not, the counter will be 2 after locking here, hence even timer expires callback do PM_RELAX, it will still not do state drop.

Reply:
The unlock has to be done on from the product code side.
It will be part of different code.